templates/modelInnerEnum.mustache

```diff
@@ -71,6 +71,11 @@
         return {{{datatypeWithEnum}}}{{^datatypeWithEnum}}{{classname}}{{/datatypeWithEnum}}.fromValue({{#isNumber}}new BigDecimal({{/isNumber}}value{{#isNumber}}){{/isNumber}});
       }
     }
+
+    public static void validateJsonElement(JsonElement jsonElement) throws IOException {
```

Comment:
This is largely a copy-paste of the `read` function above, modified to work with a `JsonElement` rather than a `JsonReader`: https://github.com/equinix-labs/metal-java/pull/163/commits/ddc37db375e481dfd1b2c317d30886dbc684a419#diff-450b10b94dc0a9d1e3ca4b6aa10dcf879ce42fddf07d6704d95c203a0398ff4dR69-R73